src/components/NcAppContent/NcAppContent.vue

```diff
@@ -192,6 +196,15 @@ export default {
 			type: Boolean,
 			default: true,
 		},
+
+		/**
+		 * Specify the `<h1>` page heading
+		 * Default is null
+		 */
+		pageHeading: {
+			type: String,
+			default: null,
+		},
```

Comment:
We don't need to mention that default is null in the description. Vue-styleguidist generates docs using the default property from the prop's definition. Developers also see it in the prop definition below.